apps/web/utils/ai/example-matches/find-example-matches.ts

```
@@ -0,0 +1,115 @@
+    .describe("The emails that match the rules prompt."),
+});
+
+export async function aiFindExampleMatches(
+  user: Pick<User, "email"> & UserAIFields,
+  gmail: gmail_v1.Gmail,
+  accessToken: string,
+  rulesPrompt: string,
+) {
+  console.log(`findExampleMatches. rulesPrompt: ${rulesPrompt}`);
+
+  const system = `You are an AI assistant specializing in email management and organization. Your task is to find example emails that match the given rules with high confidence.`;
```

Biome

[error] Do not use template literals if interpolation and special-character handling are not needed.

Unsafe fix: Replace with string literal


(lint/style/noUnusedTemplateLiteral)